Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM tiangolo/uvicorn-gunicorn-starlette:python3.8
```

Comment:
We aren't using gunicorn,  uvicorn,  or starlette anymore.

Perhaps from alpine or some other lightweight image.